src/app/(pages)/projects/_components/PastProjectCard.tsx

```diff
@@ -3,33 +3,42 @@ import React from 'react';
 import { ProjectDataType } from '@/lib/notion/projects';
 import { CldImage } from 'next-cloudinary';
 import { CardContent, CardTitle } from '@/components/ui/card';
+import LinkButton from '@/components/LinkButton';
+import { FaGithub } from 'react-icons/fa';
 
 export default function PastProjectCard({ data }: { data: ProjectDataType }) {
   return (
-    <a href={'/projects/' + data.pageId}>
-      <div className="bg-gradient-to-t from-[#0170DC] to-[#6191BC] rounded-[40px] flex flex-col max-h-[500px] p-4 outline hover:outline-white hover:outline-4 transition-all ease-in-out duration-150">
-        {data.logoUrl && (
-          <div className="flex justify-center items-center mb-4">
-            <CldImage
-              src={data.logoUrl}
-              alt={data.companyName}
-              width={250}
-              height={250}
-              className="w-48 h-48 object-fill"
-            />
+    <div className="relative">
+      <a href={'/projects/' + data.pageId}>
+        <div className="flex flex-col justify-end items-center min-h-[400px] max-h-[500px] p-4 rounded-[40px] hover:outline hover:outline-4 hover:outline-white transition-all ease-in-out duration-150 bg-gradient-to-t from-[#0170DC] to-[#6191BC]">
+          {data.logoUrl && (
+            <div className="mb-4">
+              <CldImage
+                src={data.logoUrl}
+                alt={data.companyName}
+                width={250}
+                height={250}
+                className="w-48 h-48 object-fill"
+              />
+            </div>
+          )}
+          <div className="flex flex-col items-center text-center gap-4">
+            <CardTitle className="flex md:items-center text-4xl text-white md:h-[80px]">{data.companyName}</CardTitle>
+            <CardContent className="text-2xl md:text-base text-white">
+              <p className="hidden md:line-clamp-4 h-[100px]">{data.description}</p>
+              <p className="block md:hidden">{data.productName}</p>
+            </CardContent>
           </div>
-        )}
-        <div className="flex flex-col gap-4">
-          <div>
-            <CardTitle className="text-4xl text-center text-white h-[80px] flex items-center justify-center">
-              {data.companyName}
-            </CardTitle>
-          </div>
-          <CardContent className="text-md text-center text-white">
-            <p className="h-[100px] line-clamp-4">{data.description}</p>
-          </CardContent>
         </div>
-      </div>
-    </a>
+      </a>
+      <LinkButton
+        href={data.gitHubUrl ?? '/projects'}
+        newTab={true}
+        variant="icon"
+        className="md:hidden absolute top-0 right-0"
```

Comment:
it's not that the card is too small (that card has the same height as the others)
it's because the logo has a background unlike the others which are transparent

this logo is super low resolution as well
can we replace it with a transparent version of https://yeco-canada.ca/wp-content/uploads/2024/06/cropped-1000697821.jpg ?

Reply:
> this logo is super low resolution as well can we replace it with a transparent version of https://yeco-canada.ca/wp-content/uploads/2024/06/cropped-1000697821.jpg ?

@voidranjer would we be able to get better versions of logos for some of the projects including yeco canada?

Reply:
nonetheless, will still push a change so it looks nice with all types of logos